LEAF_Request_Portal/templates/view_reports.tpl

```diff
@@ -846,32 +920,52 @@ $(function() {
                 grid.renderBody();
             }
             else {
-                var recordIDs = '';
-                for (var i in res) {
+                let recordIDs = '';
```

Comment:
I should have caught this earlier.  This isn't something to change here but to keep in mind going forward.  Try not to use concatenation inside a loop.  Instead create an array of elements.  Then outside the loop, implode them into a string.